examples/peer-and-content-routing/README.md

```diff
@@ -40,15 +43,18 @@ const node1 = nodes[0]
 const node2 = nodes[1]
 const node3 = nodes[2]
 
+node1.peerStore.addressBook.set(node2.peerId, node2.multiaddrs)
+node2.peerStore.addressBook.set(node3.peerId, node3.multiaddrs)
+
 await Promise.all([
-  node1.dial(node2.peerInfo),
-  node2.dial(node3.peerInfo)
+  node1.dial(node2.peerId),
+  node2.dial(node3.peerId)
 ])
 
 // Set up of the cons might take time
 await delay(100)
 
-const peer = await node1.peerRouting.findPeer(node3.peerInfo.id)
+const peer = await node1.peerRouting.findPeer(node3.peerId)
 
 console.log('Found it, multiaddrs are:')
 peer.multiaddrs.forEach((ma) => console.log(ma.toString()))
```

Comment:
I would change the logging here so that it does log the `/p2p` id. This ensures that the address and peer id can be associated to one another. Store separately, render together.